### PR TITLE
Rubocop config

### PR DESCRIPTION
**Pull request by micaelacole**
# Summary

Implementation of Rubocop. When the following command is executed:
`bundle exec rubocop app -R`

The files are inspected and no offenses in the application are detected by Rubocop. This is the expected behaviour of the card for it to being considered successful.

# Trello Card

https://trello.com/c/8LREWGyD

### CODE REVIEW

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Fix the conflicts and fix why it's showing 1575 files show 
image

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Documentation:
+  Enabled: false
+LineLength:
+  Max: 99
```
Comment (facucompe) [inline .rubocop.yml]: Always leave an empty line at the bottom of the file

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (facucompe) [inline Gemfile]: We don't want this comment. You can disable rubocop rules.
Add this to the **rubocop.yml**
```
Style/FrozenStringLiteralComment:
  Enabled: false
```

Comment (facucompe) [reply Gemfile]: Also remove all the comments from all files. Sorry for that :grimacing:

**Review by facucompe — CHANGES_REQUESTED** [review]

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+            # Include default devise modules.
```
Comment (nicozar95) [inline app/models/user.rb]: Why is this here? what is devise? we are installing rubocop on this PR not devise :p

**Review by nicozar95 — CHANGES_REQUESTED** [review]

**Review by nicozar95 — APPROVED** [review]

**Review by facucompe — CHANGES_REQUESTED** [review]
Remember removing the top comments from all files

**Review by facucompe — APPROVED** [review]

**Pull request merged**